test/integration/base_integration_test.h

```diff
@@ -69,7 +70,7 @@ class BaseIntegrationTest : protected Logger::Loggable<Logger::Id::testing> {
   // configuration generated in ConfigHelper::finalize.
   void skipPortUsageValidation() { config_helper_.skipPortUsageValidation(); }
   // Make test more deterministic by using a fixed RNG value.
-  void setDeterministic() { deterministic_ = true; }
+  void setDeterministicValue(uint64_t value) { deterministic_value_ = value; }
```

Comment:
I'd probably have kept this named the same thing and use a default parameter, since it seems like most call sites don't care about setting a specific parameter. This should avoid some of the churn in existing tests